server/src/test/java/io/crate/integrationtests/LuceneQueryBuilderIntegrationTest.java

```diff
@@ -425,4 +426,192 @@ public void testAnyOnNestedArray() {
             .as("query doesn't match")
             .hasRowCount(0L);
     }
+
+    @UseJdbc(0)
+    @Test
+    public void test_eq_queries_with_index_on_and_column_store_true() {
```

Comment:
Please use the `LuceneQueryBuilderTest` or `QueryTester` classes for tests instead of creating full blown integration tests.

Reply:
Oh `LuceneQueryBuilderTest` uses `Indexers`.